docs/development.md

```diff
@@ -374,6 +389,18 @@ If you want to build the documentation locally, this is how you do it:
     $ pipenv mkdocs build --config-file mkdocs.yml
     ```
 
+    _alternatively..._
+
+3.  Serve the documentation. This will spin up a
+    copy of the documentation at http://127.0.0.1:8000
+    that will automatically refresh everytime you change
+    the documentation.
+
+    ```shell-session
+    $ cd /path/to/paperless
+    $ pipenv mkdocs serve
```

Comment:
This should probably be `pipenv run` or done inside the `pipenv shell`

Reply:
This section still needs to add the `run` portion.  to the commands.  Or remove the `pipenv` portion if expected to be run in the `pipenv shell`

Reply:
My bad. Missed that one too.